src/test/java/uk/gov/hmcts/reform/pcqbackend/repository/impl/ProtectedCharacteristicsRepositoryImplTest.java

```diff
@@ -119,10 +119,11 @@ void shouldFindByCaseIdIsNullAndCompletedDateGreaterThan() {
         protectedCharacteristicsRepository.updateCase(null, "1");
```

Comment:
You need to add a test to prove that the date range query will work.

Reply:
Test has been added to prove date range query work